superwoman7/js/app.js

```diff
@@ -1,4 +1,30 @@
 var app = angular.module('tweet-timeline', ['angular-loading-bar']);
+<<<<<<< HEAD
+app.controller('tweet-search', function ($scope, $http, $sce) {
+    $scope.loading = true;
+    $scope.getTweets = function () {
+        $http.jsonp('http://api.loklak.org/api/search.json?callback=JSON_CALLBACK&q='+
+                     $scope.search).success( function (response) {
+            $scope.tweets = response.statuses;
+            for (var i = 0; i < $scope.tweets.length; ++i) {
+                $scope.tweets[i].text = $sce.trustAsHtml($scope.tweets[i].text);
+                $scope.tweets[i].user.name = $sce.trustAsHtml($scope.tweets[i].user.name);
+            }
+            $scope.searchTerm = $scope.search
+            $scope.loading = false;
+        }, function (error) {
+            alert('Oh no! An error! '+error);
+        });
+    }
+    window.addEventListener("keydown",   function flagup(event)
+     {
+              if(event.key=='Enter')
+              {
+          $scope.getTweets();
+              }
+    });
+});
+=======
 app.controller('tweet-search', function($scope, $http, $sce) {
 	$scope.loading = true;
```

Comment:
Line contains following spacing inconsistencies:
- Tabs used instead of spaces.

*SpaceConsistencyBear, severity NORMAL, section `default`.*

The issue can be fixed by applying the following patch:

```diff
--- a/superwoman7/js/app.js
+++ b/superwoman7/js/app.js
@@ -26,7 +26,7 @@
 });
 =======
 app.controller('tweet-search', function($scope, $http, $sce) {
-	$scope.loading = true;
+    $scope.loading = true;
 	$scope.getTweets = function() {
 		$http.jsonp('http://api.loklak.org/api/search.json?callback=JSON_CALLBACK&q='+$scope.search).success( function(response) {
 			$scope.tweets = response.statuses;
```